### ISSUE TITLE
[Feature request] In-app audio recorder

### ISSUE THREAD

**Issue by sigaloid** [zadam/trilium#1924]
This would be a cool addition so users could record audio notes from within the app instead of importing it.

**Comment by andrejdam** [comment]
Yep, a great idea. A speech recognition engine would also be first-class addition (there are many engines already available: https://fosspost.org/open-source-speech-recognition/)

**Comment by abc013** [comment]
Duplicate of #1389?
